app/javascript/leaflet/controls/geosearch.js

```diff
@@ -79,11 +79,11 @@ export default class GeoSearchControl extends Control {
     if (this.options.dynamic) {
       this.dynamicButton.style.display = "block";
     }
-    this.applySearch();
+    this.applySearch(false);
   }
 
   // Update the URL with the current bounding box and go back to page 1
-  applySearch() {
+  applySearch(turbo=true) {
```

Comment:
I realize it's less javascript-y, but could we do
```suggestion
  applySearch(opts={turbo: true}) {
```
so that the calls look like
```js
this.applySearch({ turbo: false })
```
which is a little more explicit about what's happening?
```js
this.applySearch(false)
```
is kinda cryptic